src/build_backend.py

```diff
@@ -87,6 +95,8 @@ def record_lines() -> Iterable[str]:
         for filename in find_sources(srcpkg=False):
             wheel.write(filename, arcname=os.path.relpath(filename, start='src'))
 
+        wheel.writestr('cockpit/data/cockpit-bridge.beipack.xz', beipack_self('cockpit.bridge:main'))
```

Comment:
Just to confirm, write*str* is okay with `Bytes`?

Reply:
I think it would have to be bytes, no?  What would it do if it's a str?

Reply:
ah,  thanks [docs](https://docs.python.org/3/library/zipfile.html#zipfile.ZipFile.writestr).

> if it is a [str](https://docs.python.org/3/library/stdtypes.html#str), it is encoded as UTF-8 first

Reply:
Ack, thanks. It's just confusingly named.